template.tpl

```diff
@@ -1138,6 +1165,236 @@ ___TEMPLATE_PARAMETERS___
         "type": "EQUALS"
       }
     ]
+  },
+  {
+    "type": "RADIO",
+    "name": "theTradeDeskTrackerOrUPixel",
+    "displayName": "Use Event Tracker ID vs. Universal Pixel ID",
+    "radioItems": [
+      {
+        "value": "tracker_id",
+        "displayValue": "Event Tracker ID"
+      },
+      {
+        "value": "upixel_id",
+        "displayValue": "Universal Pixel ID"
+      }
+    ],
+    "enablingConditions": [
+      {
+        "paramName": "tagType",
+        "paramValue": "theTradeDeskEvent",
+        "type": "EQUALS"
+      }
+    ],
+  },
+  {
+    "type": "TEXT",
+    "name": "theTradeDeskTrackerOrUPixelIDValue",
+    "displayName": "Event Tracker ID / Universal Pixel ID Value",
+    "simpleValueType": true,
+    "enablingConditions": [
+      {
+        "paramName": "tagType",
+        "paramValue": "theTradeDeskEvent",
+        "type": "EQUALS"
+      }
+    ],
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ]
+  },
+  {
+    "type": "SELECT",
+    "name": "theTradeDeskEventName",
+    "displayName": "Event Name",
+    "selectItems": [
+      {
+        "value": "addtocart",
+        "displayValue": "addtocart"
+      },
+      {
+        "value": "purchase",
+        "displayValue": "purchase"
+      },
+      {
+        "value": "viewitem",
+        "displayValue": "viewitem"
+      },
+      {
+        "value": "searchitem",
+        "displayValue": "searchitem"
+      },
+      {
+        "value": "searchcategory",
+        "displayValue": "searchcategory"
+      },
+      {
+        "value": "login",
+        "displayValue": "login"
+      },
+      {
+        "value": "messagebusiness",
+        "displayValue": "messagebusiness"
+      },
+      {
+        "value": "direction",
+        "displayValue": "direction"
+      },
+      {
+        "value": "startcheckout",
+        "displayValue": "startcheckout"
+      },
+      {
+        "value": "viewcart",
+        "displayValue": "viewcart"
+      },
+      {
+        "value": "sitevisit",
+        "displayValue": "sitevisit"
+      },
+      {
+        "value": "wishlistitem",
+        "displayValue": "wishlistitem"
+      }
+    ],
+    "notSetText": "-",
```

Comment:
This is so weird. I can't even find official documentation on this. Not commenting because I need anything, it's just really weird.

Reply:
yes it is